### PR TITLE
Disable polyfill for Helio.

### LINKED ISSUE #17814: Disable polyfill for helio
##### Remove polyfill for Helio webxr
Helio doesn't need polyfill for the up coming release. We're going to disable it.

##### Three.js version

- [x] Dev
- [ ] r109

##### Browser

- [ ] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer
- [x] Helio

##### OS

- [ ] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS
- [x] LuminOS

##### Hardware Requirements (graphics card, VR Device, ...)
Magic Leap One

### PR DESCRIPTION
**Pull request by yhuang-ml**
Disable polyfill for release 0.98.0.

Fixes #17814

### CODE REVIEW

**Comment by mrdoob** [comment]
Why not removing the polyfill instead?

**Comment by yhuangbrowser** [comment]
> Why not removing the polyfill instead?

We still need it for old versions of builds, to ensure it's backwards compatible.

**Comment by mrdoob** [comment]
Ah, okay.

**Comment by mrdoob** [comment]
Thanks!

File: examples/js/vr/HelioWebXRPolyfill.js [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 if ( /(Helio)/g.test( navigator.userAgent ) && "xr" in navigator ) {
 
+	if ('isSessionSupported' in navigator.xr) {
+        return;
+    }
+
```
Comment (takahirox) [inline examples/js/vr/HelioWebXRPolyfill.js]: It seems this change causes #17837 because return is used not in function.

Comment (yhuangbrowser) [reply examples/js/vr/HelioWebXRPolyfill.js]: Thanks for heads up. Taking a look.

**Pull request merged**